client/components/happychat/notices.jsx

```diff
@@ -36,16 +41,19 @@ export class Notices extends Component {
 		}
 
 		switch ( connectionStatus ) {
-			case 'uninitialized':
+			case HAPPYCHAT_CONNECTION_STATUS_UNINITIALIZED:
 				return translate( 'Waiting to connect you with a Happiness Engineer…' );
-			case 'connecting':
+			case HAPPYCHAT_CONNECTION_STATUS_CONNECTING:
 				return translate( 'Connecting you with a Happiness Engineer…' );
-			case 'reconnecting':
-			// Fall through to the same notice as `disconnected`
-			case 'disconnected':
+			case HAPPYCHAT_CONNECTION_STATUS_RECONNECTING:
+			case HAPPYCHAT_CONNECTION_STATUS_DISCONNECTED:
 				return translate(
 					"We're having trouble connecting to chat. Please bear with us while we try to reconnect…"
 				);
+			case HAPPYCHAT_CONNECTION_STATUS_UNAUTHORIZED:
+				return translate(
+					'Your user is not authorized to chat at the moment. Would you mind contacting support by other means?'
```

Comment:
I personally felt a bit confused when I read "Your user" here. Maybe "Your account" or simply "You are not authorized ..." would be easier to understand? I'm not quite confident here so I'm labeling this for copy review :)

Reply:
Improved messaging in e7c2691

Reply:
Updated text in adf6707 as discussed with Krista in p1509538277000157-slack-editorial

Go to merge this in.